<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>363</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Bahnschrift SemiLight</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="pageDrag">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="labelDrag">
          <property name="text">
           <string>Drag and Drop Image</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageSize">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupCrop">
          <property name="title">
           <string>Crop</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>7</number>
           </property>
           <property name="leftMargin">
            <number>25</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>25</number>
           </property>
           <property name="bottomMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QRadioButton" name="rectangle">
             <property name="text">
              <string>Rectangle</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="self">
             <property name="text">
              <string>Self-Selected</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDialogButtonBox" name="cropCheck">
             <property name="standardButtons">
              <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupSize">
          <property name="font">
           <font>
            <family>Bahnschrift SemiLight</family>
           </font>
          </property>
          <property name="title">
           <string>Resize</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <property name="horizontalSpacing">
            <number>30</number>
           </property>
           <property name="verticalSpacing">
            <number>15</number>
           </property>
           <property name="leftMargin">
            <number>25</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>25</number>
           </property>
           <property name="bottomMargin">
            <number>10</number>
           </property>
           <item row="1" column="0">
            <widget class="QLabel" name="percentage">
             <property name="text">
              <string>Percentage(%)</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSlider" name="sizeSlider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QDialogButtonBox" name="sizeCheck">
             <property name="standardButtons">
              <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageBlur">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGroupBox" name="groupBlur">
          <property name="title">
           <string>Blur</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <property name="spacing">
            <number>20</number>
           </property>
           <property name="leftMargin">
            <number>20</number>
           </property>
           <property name="topMargin">
            <number>25</number>
           </property>
           <property name="rightMargin">
            <number>20</number>
           </property>
           <property name="bottomMargin">
            <number>25</number>
           </property>
           <item>
            <widget class="QWidget" name="widget" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <property name="leftMargin">
               <number>10</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QRadioButton" name="box">
                <property name="text">
                 <string>Box Filter</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="median">
                <property name="text">
                 <string>Median Filter</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="gaussian">
                <property name="text">
                 <string>Gaussian Filter</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="bilateral">
                <property name="text">
                 <string>Bilateral Filter</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_5" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Kernel  </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="kernel"/>
              </item>
              <item>
               <widget class="QDialogButtonBox" name="blurCheck">
                <property name="standardButtons">
                 <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageColorModel">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupColorM">
          <property name="title">
           <string>cvtColor</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <property name="spacing">
            <number>20</number>
           </property>
           <property name="leftMargin">
            <number>20</number>
           </property>
           <property name="topMargin">
            <number>25</number>
           </property>
           <property name="rightMargin">
            <number>20</number>
           </property>
           <property name="bottomMargin">
            <number>25</number>
           </property>
           <item>
            <widget class="QWidget" name="widget_2" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <property name="spacing">
               <number>7</number>
              </property>
              <property name="leftMargin">
               <number>10</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QRadioButton" name="gray">
                <property name="text">
                 <string>Grqy model</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="hsv">
                <property name="text">
                 <string>HSV model</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="hsl">
                <property name="text">
                 <string>HSL model</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="ycrcb">
                <property name="text">
                 <string>YCrCb model</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QDialogButtonBox" name="colorMCheck">
             <property name="standardButtons">
              <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageChannel">
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <widget class="QGroupBox" name="groupChannel">
          <property name="title">
           <string>Channel</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="rightMargin">
            <number>16</number>
           </property>
           <item>
            <widget class="QWidget" name="widget_3" native="true">
             <layout class="QFormLayout" name="formLayout_2">
              <property name="horizontalSpacing">
               <number>30</number>
              </property>
              <property name="leftMargin">
               <number>25</number>
              </property>
              <property name="rightMargin">
               <number>40</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="red">
                <property name="text">
                 <string>Red</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSlider" name="redSlider">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="green">
                <property name="text">
                 <string>Green</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QSlider" name="greenSlider">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="blue">
                <property name="text">
                 <string>Blue</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QSlider" name="blueSlider">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="contrst">
                <property name="text">
                 <string>Contrast</string>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QSlider" name="contrastSlider">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item row="8" column="0">
               <widget class="QLabel" name="brightness">
                <property name="text">
                 <string>Brightness</string>
                </property>
               </widget>
              </item>
              <item row="8" column="1">
               <widget class="QSlider" name="brightnessSlider">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="5" column="0">
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="3" column="0">
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="0">
               <spacer name="verticalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="9" column="0">
               <spacer name="verticalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QDialogButtonBox" name="channelCheck">
             <property name="standardButtons">
              <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageSharpen">
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <item>
         <widget class="QGroupBox" name="groupSharpen">
          <property name="title">
           <string>Sharpen</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_14">
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>25</number>
           </property>
           <property name="bottomMargin">
            <number>18</number>
           </property>
           <item>
            <widget class="QWidget" name="widget_4" native="true">
             <layout class="QVBoxLayout" name="verticalLayout">
              <property name="leftMargin">
               <number>30</number>
              </property>
              <property name="rightMargin">
               <number>140</number>
              </property>
              <property name="bottomMargin">
               <number>10</number>
              </property>
              <item>
               <widget class="QRadioButton" name="general">
                <property name="text">
                 <string>General type</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="generalSlider">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="laplacian">
                <property name="text">
                 <string>Laplacian Operator</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="laplacianSlider">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QDialogButtonBox" name="sharpenCheck">
             <property name="standardButtons">
              <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageColor">
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <item>
         <widget class="QGroupBox" name="groupColor">
          <property name="title">
           <string>Color Effects</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_16">
           <property name="spacing">
            <number>15</number>
           </property>
           <property name="leftMargin">
            <number>20</number>
           </property>
           <property name="topMargin">
            <number>20</number>
           </property>
           <property name="rightMargin">
            <number>20</number>
           </property>
           <property name="bottomMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="QScrollArea" name="scrollArea">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>98</width>
                <height>28</height>
               </rect>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QDialogButtonBox" name="colorCheck">
             <property name="standardButtons">
              <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageSpecial"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="Open"/>
    <addaction name="Save"/>
    <addaction name="Undo"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Resize(&amp;R)</string>
    </property>
    <addaction name="Size"/>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="title">
     <string>Image(&amp;I)</string>
    </property>
    <addaction name="Blur"/>
    <addaction name="ColorModel"/>
    <addaction name="Channel"/>
    <addaction name="Sharpen"/>
   </widget>
   <widget class="QMenu" name="menuEffect">
    <property name="title">
     <string>Effect(&amp;E)</string>
    </property>
    <addaction name="Color"/>
    <addaction name="Special"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuImage"/>
   <addaction name="menuEffect"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="Open">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="Save">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="Undo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="Size">
   <property name="text">
    <string>Size</string>
   </property>
  </action>
  <action name="Blur">
   <property name="text">
    <string>Blur</string>
   </property>
  </action>
  <action name="ColorModel">
   <property name="text">
    <string>Color model</string>
   </property>
  </action>
  <action name="Channel">
   <property name="text">
    <string>Channel</string>
   </property>
  </action>
  <action name="Sharpen">
   <property name="text">
    <string>Sharpen</string>
   </property>
  </action>
  <action name="Color">
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="Special">
   <property name="text">
    <string>Special</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
